src/Module/Profile/Index.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+namespace Friendica\Module\Profile;
+
+use Friendica\BaseModule;
+use Friendica\Content\Feature;
+use Friendica\Content\ForumManager;
+use Friendica\Content\Nav;
+use Friendica\Content\Text\BBCode;
+use Friendica\Content\Text\HTML;
+use Friendica\Core\Hook;
+use Friendica\Core\Protocol;
+use Friendica\Core\Renderer;
+use Friendica\Core\Session;
+use Friendica\Core\System;
+use Friendica\Database\DBA;
+use Friendica\DI;
+use Friendica\Model\Contact;
+use Friendica\Model\Profile;
+use Friendica\Model\Term;
+use Friendica\Model\User;
+use Friendica\Module\Security\Login;
+use Friendica\Network\HTTPException;
+use Friendica\Protocol\ActivityPub;
+use Friendica\Util\DateTimeFormat;
+use Friendica\Util\Temporal;
+
+require_once 'boot.php';
```

Comment:
Is this really necessary anymore? Isn't `boot.php` always loaded?

Reply:
You're right.